runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -1316,9 +1316,34 @@ MM_IncrementalGenerationalGC::partialGarbageCollectUsingCopyForward(MM_Environme
 
 	UDATA desiredCompactWork = _schedulingDelegate.getDesiredCompactWork();
 	UDATA estimatedSurvivorRequired = _copyForwardDelegate.estimateRequiredSurvivorBytes(env);
+
+	MM_GlobalAllocationManagerTarok *allocationmanager = (MM_GlobalAllocationManagerTarok *)_extensions->globalAllocationManager;
+	UDATA freeRegions = allocationmanager->getFreeRegionCount();
+	double estimatedReguiredSurvivorRegions = _schedulingDelegate.getAverageSurvivorSetRegionCount();
+	MM_GCExtensions* extensions = MM_GCExtensions::getExtensions(env);
+	/* Adjust estimatedReguiredSurvivorRegions if extensions->fvtest_forceCopyForwardHybridRatio is set(for testing purpose) */
+	if ((0 != extensions->fvtest_forceCopyForwardHybridRatio) && (100 >= extensions->fvtest_forceCopyForwardHybridRatio)) {
+		estimatedReguiredSurvivorRegions = estimatedReguiredSurvivorRegions * (100 - extensions->fvtest_forceCopyForwardHybridRatio) / 100;
+	}
+	j9tty_printf(PORTLIB, "partialGarbageCollectUsingCopyForward freeRegions=%zu, estimatedReguiredSurvivorRegions=%f\n", freeRegions, estimatedReguiredSurvivorRegions);
```

Comment:
remnant print here

Reply:
the printfs are for my local debug, I will remove them.